structure/mysql_vm_repository.go

```
@@ -0,0 +1,184 @@
+	tx, err := r.DB.Begin()
+	if err != nil {
+		log.Error("Failed to start transaction: %v", err)
+		return nil, nil, err
+	}
+	defer tx.Rollback()
+
+	ctx, cancel := context.WithTimeout(context.Background(), 3*time.Second)
+	defer cancel()
+
+	var rows *sql.Rows
+	rows, err = tx.QueryContext(ctx, "SELECT info.uuid, loc.core, info.inst_ip, info.guac_pass, info.inst_vcpu, info.inst_mem, info.inst_disk FROM inst_loc loc JOIN inst_info info ON loc.uuid = info.uuid")
```

golangci-lint (2.11.4)

[error] rows.Err must be checked

(rowserrcheck)